src/nxdoc/nuxeo-add-ons/nuxeo-aspera-connector.md

```diff
@@ -14,24 +14,399 @@ tree_item_index: 1100
 
 {{! excerpt}}
 The [Nuxeo Aspera](https://connect.nuxeo.com/nuxeo/site/marketplace/package/nuxeo-aspera) connector
-enables users to download/upload binaries with Aspera.
+enables users to download/upload binaries with Aspera.  It is designed to be used with **Aspera On Cloud** instances, however, if you use a **self-hosted Aspera** instance, it is still possible you will be able to use the Nuxeo Aspera Connector.  Contact your Aspera team for more information and assistance.
 {{! /excerpt}}
 
-## Requirements
+## Version 3.0.0
+### Requirements
+
+- Aspera Desktop Client - you will be prompted to install necessary versions when using the Connector.
+- Nuxeo Server LTS 2019 (**10.10, HF 31, at least**) with access to AWS S3 Storage along with the [Amazon S3 Online Storage plugin](https://connect.nuxeo.com/nuxeo/site/marketplace/package/amazon-s3-online-storage?version=11.2.13)
+- IBM Aspera on Cloud Subscription (see note above about Aspera self-hosted)
+
+
+{{#> callout type='warning'}}
+If you are **upgrading from a previous version** of the Nuxeo Aspera connector, please see the [upgrade notes](https://doc.nuxeo.com/nxdoc/nuxeo-aspera-connector/nuxeo-aspera-upgrade-notes/) for critical information related to potential breaking code.
+{{/callout}}
```

Comment:
The release not is not yet available, isn't it ?

Reply:
not 3.0 but previous ones, yes.